composer.json

```diff
@@ -130,7 +125,7 @@
     ],
     "auto-scripts": {
       "cache:clear": "symfony-cmd",
-      "assets:install %PUBLIC_DIR%": "symfony-cmd"
```

Comment:
Why was this removed? It is default when installing Symfony, but might be unnecessary (?)

Reply:
Good point, it seemed to be in the way and looked to cause a malfunction. But this is indeed the standard. And in this case functioned as a red herring.. 

Reverted.